src/org/zaproxy/zap/view/ZapSupportPanel.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2017 The ZAP Development Team
+ *  
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.view;
+
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.Insets;
+
+import javax.swing.JPanel;
+import javax.swing.JTextArea;
+
+import org.apache.commons.lang.WordUtils;
+import org.zaproxy.zap.utils.ZapSupportUtils;
+
+public class ZapSupportPanel extends JPanel {
+
+	private static final long serialVersionUID = 1L;
+	private static final String NEWLINE = System.lineSeparator();
+	private static final String PRODUCT = ZapSupportUtils.getProductName();
+	private static final String VERSION = ZapSupportUtils.getVersion();
+	private static final String ZAP_HOME = ZapSupportUtils.getZapHomeDirectory();
+	private static final String OS = ZapSupportUtils.getOperatingSystem();
+	private static final String JAVA_VERSION = ZapSupportUtils.getJavaVersionVendor();
+	private static final String INSTALLED_ADDONS = ZapSupportUtils.getInstalledAddons();

```

Comment:
This (`getInstalledAddons()`) needs to be called each time the dialogue is shown otherwise it might show outdated info (i.e. add-ons can be dynamically un/installed).